docs/docs-content/clusters/public-cloud/azure/aks.md

```diff
@@ -74,7 +74,22 @@ explains how you can create an Azure AKS cluster managed by Palette.
 
 6.  Under **Managed Kubernetes**, select **Azure AKS** and select your Azure AKS cluster profile.
 
-7.  If you want to configure Pod and Service CIDR, populate the following configuration template and add the
+7.  Palette displays the cluster profile layers. Review the profile layers and customize parameters as desired in the
+    YAML files that display when you select a layer.
```

Comment:
Do we need to say that they need to click on Values to see the YAML?

Reply:
Good callout.